src/App.js

```diff
@@ -1,25 +1,29 @@
-import logo from './logo.svg';
+
 import './App.css';
+import HomePage from './components/HomePage';
+import Beer from './components/Beer';
+import NewBeer from './components/NewBeer';
+import RandomBeer from './components/RandomBeer';
+import { Routes, Route} from 'react-router-dom';
+import Header from './components/Header';
+
+
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+   
+    <div>
+  <Header/>
```

Comment:
Because you have `<Header />` here you could remove from the other components :)